core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Ck editor copy toolbar component.
+ */
+
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { CkEditorCopyContentService } from
+  'components/ck-editor-helpers/ck-editor-copy-content-service';
+
+
+@Component({
+  selector: 'ck-editor-copy-toolbar',
+  templateUrl: './ck-editor-copy-toolbar.component.html'
+})
+export class CkEditorCopyToolbarComponent {
+  constructor(private ckEditorCopyContentService: CkEditorCopyContentService) {
+    ckEditorCopyContentService.copyModeActive = false;
+  }
+
+  toggleToolActive() {
+    this.ckEditorCopyContentService.toggleCopyMode();
+
+    if (this.ckEditorCopyContentService.copyModeActive) {
+      document.body.style.cursor = 'copy';
```

Comment:
We directly use `document` in angualr there isn't any wrapper like $document in Angular? (I'm not familiar with Angular)

Reply:
Not in Angular2+, I believe

Reply:
You are right, it is ```import { DOCUMENT } from '@angular/common';```. Will fix